super-team-picker/db/migrations/20191215122550_AddImageToCohort.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = function(knex) {
+    return knex.schema.table("cohorts", table => {
+        table.string("imageUrl"); // Add Column 'imageUrl' VARCHAR(255)
```

Comment:
Use `table.text` instead of `string` to use a data type that accepts a length that's greater than 255, since image urls can get long.